website/pages/docs/builders/vmware/vsphere-iso.mdx

```diff
@@ -136,6 +155,23 @@ from the datastore. Example:
 ],
 ```
 
+</Tab>
+<Tab heading="HCL2">
+
+```hcl
+iso_urls = [
+  "win10.iso",
+  "http://example.org/isos/win10.iso"
+]
+
+iso_paths = [
+    "[] /usr/lib/vmware/isoimages/windows.iso"
```

Comment:
Did you mean:

```suggestion
    "/usr/lib/vmware/isoimages/windows.iso"
```

Reply:
The iso_paths has this format they can be like `"[datastore1] /usr/lib/vmware/isoimages/windows.iso"`